flowcraft/generator/components/typing.py

```diff
@@ -161,4 +161,101 @@ def __init__(self, **kwargs):
         ]
 
 
+class Seqsero2Reads(Process):
+    """SeqSero2 for reads process template interface
 
+    This process is set with:
+
+        - ``input_type``: fastq
+        - ``output_type``: None
+        - ``ptype``: typing
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = None
+
+        self.directives = {
+            "seqsero2_reads": {
+                "cpus": 1,
+                "memory": "{ 1.GB * task.cpus * task.attempt }",
+                "container": "ummidock/seqsero2",
+                "version": "alpha-test-1",
+                "cache": "false",
+                "scratch": "true"
+            }
+        }
+
+
+class Seqsero2Assembly(Process):
+    """SeqSero2 for assembly process template interface
+
+    This process is set with:
+
+        - ``input_type``: fasta
+        - ``output_type``: None
+        - ``ptype``: typing
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fasta"
+        self.output_type = None
+
+        self.directives = {
+            "seqsero2_assembly": {
+                "cpus": 1,
+                "memory": "{ 1.GB * task.cpus * task.attempt }",
+                "container": "ummidock/seqsero2",
+                "version": "alpha-test-1",
+                "cache": "false",
+                "scratch": "true"
+            }
+        }
+
+
+class StxSeqtypingReads(Process):
```

Comment:
Same comment as before. Why state that is is for reads in the component name? I would change to "StxSeqTyping".

Reply:
Replied above

Reply:
Done